src/main/java/seedu/address/model/person/Person.java

```diff
@@ -25,11 +25,12 @@
     private final Address address;
     private final Set<ClientSource> clientSources = new HashSet<>();
     private final Note note;
+    private final boolean isArchive;
     private final Priority priority;
 
-
     /**
-     * Only name and tags need to be non-null.
+     * Only name, tags need to be non-null.
```

Comment:
What do you mean by tags?

Reply:
I think if it means clientsource, then clientsource should be non null. 

Also, priority should be non null but i should be the one writing that XD